bundle.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM scratch
```

Comment:
Shall we remove this file? since the generated `bundle` is not used.

Reply:
This is a generated file. I don't really know what is it for. I think we can keep it for now. ART will not process this file.